client/eslint.config.mjs

```diff
@@ -0,0 +1,16 @@
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+});
+
+const eslintConfig = [
+  ...compat.extends("next/core-web-vitals", "next/typescript"),
```

Comment:
Add the rules that we used in Channel Surfers:

```js
        rules: {
            camelcase: [
                'error',
                {
                    ignoreImports: true,
                },
            ],
            eqeqeq: 'error',
            'no-useless-assignment': 'warn',
            'consistent-return': 'error',
            'dot-notation': 'error',
            'no-unneeded-ternary': 'warn',
            'object-shorthand': 'error',
            'prefer-const': 'warn',
            'prefer-destructuring': 'error',
            'no-useless-rename': 'error',
            'no-cond-assign': 'error',
            '@typescript-eslint/no-unused-vars': [
                'error', // Change to 'error' if you want it to be an error
                {
                    args: 'all',
                    argsIgnorePattern: '^_',
                    caughtErrors: 'all',
                    caughtErrorsIgnorePattern: '^_',
                    destructuredArrayIgnorePattern: '^_',
                    varsIgnorePattern: '^_',
                    ignoreRestSiblings: true,
                },
            ],
        },
```

Reply:
added